src/frontend/main.go

```diff
@@ -157,6 +184,17 @@ func initStats(log logrus.FieldLogger, exporter *stackdriver.Exporter) {
 	}
 }
 
+func initMeter(log logrus.FieldLogger) *push.Controller {
+	pusher, err := metricstdout.InstallNewPipeline(metricstdout.Config{
+		Quantiles:   []float64{0.5, 0.9, 0.99},
+		PrettyPrint: false,
+	})
+	if err != nil {
+		log.Panicf("failed to initialize metric stdout exporter %v", err)
+	}
+	return pusher
+}
+
 func initStackdriverTracing(log logrus.FieldLogger) {
```

Comment:
This method name is not correct anymore :)

Reply:
changed to `initOpenCensus` :)